common/jenkins-agents/python/docker/Dockerfile

```diff
@@ -5,45 +5,42 @@ LABEL maintainer="Gerard Castillo <gerard.castillo@boehringer-ingelheim.com>"
 ARG nexusHost
 ARG nexusAuth
 
-ENV PYTHON_VERSION=3.6.10
```

Comment:
I liked having these env variables.

Reply:
yep, that was removed in favour of handling the versions by the package offered from redhat/centos

I agree that this current proposal might lead to less flexibility so I will continue working on this and test a new python compiled approach